openstack/keppel-trivy/values.yaml

```
@@ -0,0 +1,47 @@
+global:
+  tld: DEFINED_IN_VALUES_FILE
+  region: DEFINED_IN_VALUES_FILE
+  registry: DEFINED_IN_VALUES_FILE
+
+owner-info:
+  support-group: containers
+  service: keppel
+  maintainers:
+    - Stefan Majewsky
+    - Sandro Jäckel
+    - Falk Reimann
+    - Muhammad Talal Anwar
+  helm-chart-url: https://github.com/sapcc/helm-charts/tree/master/openstack/keppel-trivy
+
+trivy:
+  image_version: "0.40.0"
+
+redis:
+  persistence:
+    enabled: true
+    accessMode: ReadWriteMany
```

We don't support ReadWriteMany on our Cinder storage class, so this gets downgraded to ReadWriteOnce by the CSI.
```suggestion
    accessMode: ReadWriteOnce
```